obci/interfaces/bci/p300_MD/helper_functions.py

```diff
@@ -0,0 +1,503 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# based on obci.analysis.p300.analysis_offline
+# Marian Dovgialo
+from scipy import *
```

Comment:
tell it to the author of this file https://github.com/BrainTech/openbci/blob/master/obci/analysis/p300/analysis_offline.py I needed to use and modify functions from there, so I copied it, it was when I was making first classifier prototypes outside OBCI